polaris-react/src/components/VideoThumbnail/VideoThumbnail.scss

```
@@ -69,7 +69,7 @@ $progress-bar-height: 6px;
   margin-left: var(--p-space-400);
   border-radius: var(--p-border-radius-2);
   color: var(--p-color-text-on-color);
```

```suggestion
  color: var(--p-color-video-thumbnail-play-button-text-on-bg-fill);
```

Icon and text should use the token above